### ISSUE TITLE
Error of "Source line too long" when compiling HEMCO code in CAM with nvhpc/23.1 compiler 

### ISSUE THREAD

**Issue by sjsprecious** [geoschem/HEMCO#233]
### Name and Institution (Required)

Name: Jian Sun
Institution: NCAR

### Confirm you have reviewed the following documentation

- [x] [Support guidelines](https://hemco.readthedocs.io/en/stable/reference/SUPPORT.html)
- [x] [User manuals](https://hemco.readthedocs.io/en/stable/geos-chem-shared-docs/supplemental-guides/related-docs.html)
- [x] [Debugging GEOS-Chem and HEMCO errors](https://hemco.readthedocs.io/en/stable/geos-chem-shared-docs/supplemental-guides/debug-guide.html)
- [x] [Current and past Github issues](https://github.com/geoschem/HEMCO/issues)

### Description of your issue or question

When I tried to build cam6_3_119 (with HEMCO tag: `hemco-cesm1_2_0_hemco3_6_2_cesm`) on Derecho with the nvhpc/23.1 compiler, the compilation failed with the error message below:
```
NVFORTRAN-S-0285-Source line too long (/glade/derecho/scratch/sunjian/CESM/components/cam/src/hemco/HEMCO/src/Core/hcoio_read_std_mod.F90: 65)
NVFORTRAN-S-0023-Syntax error - unbalanced parentheses (/glade/derecho/scratch/sunjian/CESM/components/cam/src/hemco/HEMCO/src/Core/hcoio_read_std_mod.F90: 65)
```
The culprit is that the `GC_72_EDGE_SIGMA` array in the `hcoio_read_std_mod.F90` code (indicated by the line number above) is too long. The solution is to break the `GC_72_EDGE_SIGMA` array into multiple lines and use the continuation character.

More details could be found at https://github.com/ESCOMP/CAM/issues/871.

**Comment by jimmielin** [comment]
Pull request submitted to https://github.com/geoschem/HEMCO/pull/234

**Comment by yantosca** [comment]
PR #234 has been merged into the `dev/3.7.1` branch.  We can now close this issue.
